development/kops/auth.sh

```diff
@@ -0,0 +1,12 @@
+unset AWS_ACCESS_KEY_ID
+unset AWS_SECRET_ACCESS_KEY
+unset AWS_SESSION_TOKEN
+export AWS_DEFAULT_PROFILE=default
+export AWS_PROFILE=default
+assume_test_role_output=`aws sts assume-role --role-arn $TEST_ROLE_ARN --role-session-name test-role-session`
+export AWS_ACCESS_KEY_ID=`echo $assume_test_role_output|jq -r .Credentials.AccessKeyId`
```

Comment:
I thought we were trying to avoid doing this and instead use config file no? Based on what Micah showed us about how it might not be an SDK issue, unless we concluded that it's still a Kops issue.

Reply:
So far i could not get it to work with prow.